2_process/src/create_lake_task_plan.R

```diff
@@ -34,10 +37,10 @@ create_lake_tasks <- function(task_df_fn, log_folder){
   )
   
   calculate_pb0_toha <- scipiper::create_task_step(
```

Comment:
Is it worth changing this object name to be more generic?

Reply:
same answer as before. updating and we'll see what happens.